Doc/library/venv.rst

```diff
@@ -122,7 +122,8 @@ creation according to their needs, the :class:`EnvBuilder` class.
 
     * ``prompt`` -- a String to be used after virtual environment is activated
       (defaults to ``None`` which means directory name of the environment would
-      be used).
+      be used). If the special string ``"."` is provided, the basename of the
```

Comment:
Is there an extra or missing backtick on the formatting for "."?

Reply:
Ah yes, well spotted!